docs/4-user-guide/2-zarf-packages/1-zarf-packages.md

```diff
@@ -4,72 +4,69 @@ sidebar_position: 1
+Zarf Packages are built while connected to the internet and to whatever is hosting the dependencies your package definition defined. This allows for the retrieval and storage of defined dependencies within the archive. As a result, the package can be deployed on disconnected systems without requiring a connection to the outside world.
 
-The `zarf.yaml` file, which the package builds from, defines declarative instructions on how the capabilities of the package should be deployed. The declarative nature of the package means everything is represented by code and automatically runs as it is configured, instead of having to give manual steps that might not be reproducible on all systems.
+The `zarf.yaml` file defines declarative instructions for the deployment of package capabilities, which are automatically executed and represented by code. This ensures reproducibility across different systems without the need for manual configuration.
 
-Zarf Packages are made up of functionality blocks called components which are described more on the [Zarf Components page](./2-zarf-components.md). These components can be optional, giving more flexibility to how packages can be used.
+Zarf Packages consist of functionality blocks, known as components. Components can be optional, providing greater flexibility in package usage. Further details on components can be found on the [Zarf Components](./2-zarf-components.md) page.
 
-## Deploying on to Airgapped Systems
+## Deploying onto Air-gapped Systems
 
-Zarf packages are built with all the dependencies necessary being included within the package itself, this is important when deploying to air-gapped systems. Since there is no need for an outbound connection to the internet, these packages become highly distributable and can be run on the edge, embedded systems, secure cloud, data centers, or even in a local environment. When deploying a package onto a cluster, the dependencies of the cluster (which were included in the package itself when it was created) are pushed into a docker registry and git server that Zarf stands up on the air-gapped system. This way later steps can use the dependencies as they are needed.
+Zarf Packages are built to include all necessary dependencies within the package itself, making it particularly useful for deploying to air-gapped systems. This eliminates the need for outbound internet connectivity, making the packages easily distributable and executable on a variety of systems, including edge, embedded systems, secure cloud, data centers, or local environments.
+
+When deploying a package onto a cluster, the dependencies of the cluster, which were included within the package upon creation, are automatically pushed into a Docker registry and Git server created by Zarf on the air-gapped system. This enables later steps to utilize these dependencies as needed.
```

Comment:
```suggestion
When deploying a package onto a cluster, the dependencies contained in each component are automatically pushed into a Docker registry and/or Git server created by or known to Zarf on the air-gapped system. This enables any later steps to utilize these dependencies as needed.
```

Reply:
A bit more clarification - you don't need to use Zarf's services and can use your own